src/extension.ts

```diff
@@ -202,8 +202,7 @@ class ExtensionManager implements vscode.Disposable {
   // because currently there is no way of reading a context variable
   // like cmake:enableFullFeatureSet, to apply the OR operation on it.
   private _enableFullFeatureSetOnWorkspace = false;
-  public enableFullFeatureSet(fullFeatureSet: boolean, folder: vscode.WorkspaceFolder) {
-    this.getFolderContext(folder).ignoreCMakeListsMissing = !fullFeatureSet;
+  public enableFullFeatureSet(fullFeatureSet: boolean) {
```

Comment:
Just for my benefit, it looks like we can't do the enable/disable per workspace in multi-root. Is it ever the case that one workspace might not want it enabled and end up disabling it for the whole workspace?

Reply:
Once I read your latest comments I though so too and I removed much of the functionality but in the end with testing I brought it all back. If the last folder that is attempted to be configured (via a configureAll for example) prompts and the user chooses to ignore, then without the workspace folder by folder check we end up with the UI disabled.

Reply:
If we track all of the "enable" states for all folders in an array `[true, false, true]` (just an example -- it doesn't need to be an array), then whenever this array changes, we set enabled to true if any folder is true, and false if all folders are false.